plugins/services/src/js/components/HighOrderServiceConfiguration.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import Pod from "../structs/Pod";
+import PodConfigurationContainer
+  from "../containers/pod-configuration/PodConfigurationContainer";
+import Service from "../structs/Service";
+import ServiceConfigurationContainer
+  from "../containers/service-configuration/ServiceConfigurationContainer";
+
+const HighOrderServiceConfiguration = props => {
```

Comment:
Please use old school functions for function components to make sure the the context of the surrounding module don't get mixed with the component. For details regarding arrow functions please see: http://stackoverflow.com/questions/22939130/when-should-i-use-arrow-functions-in-ecmascript-6